grc/workflows/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# Copyright 2024 Free Software Foundation, Inc.
```

Comment:
:thinking:

Reply:
I'm removing this. No one should claim copyright on a single line of `include(GrPython)`